doc/manual/manual.tex

```diff
@@ -8550,6 +8550,44 @@ \subsubsection{Melt migration in a 2D mid-ocean ridge model}
         effects might also play a role).
 \end{itemize}
 
+\subsubsection{Polydiapirism}
+\label{sec:cookbooks-polydiapirism}
+
+We consider a three-layer viscous Rayleigh-Taylor instability proposed by Weinberg and Schmeling \cite{wesc92} 
+and we focus in what follows on the case II of this publlication.
+The domain is a 2D Cartesian box of size $2.24\times 1$. Gravity is Earth-like.
+Boundary conditions are free-slip on the sides and top and no-slip at the bottom.
+All three layers are initially horizontal. The top layer (fluid 1) has a thickness of 0.75, a viscosity $\eta_1=100$ and a density $\rho_1=100$. The middle layer (fluid 2) has a thickness 0.125 with $\rho_2=90$ and $\mu_2=1$.
```

Comment:
Does the gravity have regular units (m/s^2)? Would it be possible to make all units either dimensional or non dimensional? Not critical to do so, but might be worth adding in sentence noting the difference.